lua/cmp/core.lua

```diff
@@ -119,6 +119,7 @@ core.on_keymap = function(self, keys, fallback)
     local is_printable = char.is_printable(string.byte(chars, 1))
     self:confirm(e, {
       behavior = is_printable and 'insert' or 'replace',
+      commit_chars = chars,
```

Comment:
could you change the name to commit_character?

Reply:
Sure, no problem